### PR TITLE
Fix show password toggle; fixes #933

### LINKED ISSUE #933: Data Source Selection window - show password button no longer working
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-IME
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
The **show** password button no longer shows the password when clicked

### Steps to Reproduce

- pop data source selection window
- click **show** button and see that nothing happens - expected is that password becomes visible and is no longer masked

image

### Logs

  - [x] There are no logs regarding the issue
  
### System Configuration

- COMET version:
  - [x] COMET-IME: 9.0.0.1

### PR DESCRIPTION
**Pull request by omaratrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description

Fix the show password button in the login screen.

### CODE REVIEW

File: CDP4ShellDialogs/Views/DataSourceSelection.xaml [hunk 1]
```diff
@@ -7,7 +7,7 @@
              xmlns:dxlc="http://schemas.devexpress.com/winfx/2008/xaml/layoutcontrol"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:navigation="clr-namespace:CDP4Composition.Navigation;assembly=CDP4Composition"
-             xmlns:views="clr-namespace:CDP4Composition.Views;assembly=CDP4Composition"
+             xmlns:views="clr-namespace:CDP4Composition.Views;assembly=CDP4Composition" xmlns:dxe1="clr-namespace:DevExpress.Xpf.Core;assembly=DevExpress.Xpf.Core.v20.2"
```
Comment (alexatrhea) [inline CDP4ShellDialogs/Views/DataSourceSelection.xaml:10]: This is not right

**Review by alexatrhea — CHANGES_REQUESTED** [review]

File: CDP4ShellDialogs/Views/DataSourceSelection.xaml [hunk 2]
```diff
@@ -116,10 +117,11 @@
                                                   Visibility="{Binding ElementName=ShowPasswordBox,
                                                        Path=IsChecked,
                                                        Converter={dx:BooleanToVisibilityConverter}}" />
-                                    <dxe:CheckEditBox x:Name="ShowPasswordBox"
+                                    <dxe1:SimpleButton x:Name="ShowPasswordBox"
```
Comment (samatrhea) [inline CDP4ShellDialogs/Views/DataSourceSelection.xaml]: the new namespace needs to be removed as @alexatrhea suggested and this needs to be `<dx:SimpleButton`

File: CDP4ShellDialogs/Views/DataSourceSelection.xaml (hunk 1 above)
Comment (omaratrhea) [reply CDP4ShellDialogs/Views/DataSourceSelection.xaml]: References changed

**Review by samatrhea — CHANGES_REQUESTED** [review]
please remove the new inserted namespace and use `<dx:SimpleButton` instead. While testing i also noticed that when we click on "show" and the password is shown the text on the button stays as `Show`. Would be nice that the caption/text toggles to "hide" when the passowrd is shown

File: CDP4ShellDialogs/Views/DataSourceSelection.xaml (hunk 2 above)
Comment (omaratrhea) [reply CDP4ShellDialogs/Views/DataSourceSelection.xaml]: namespace removed and changed to dx

**Review by alexatrhea — APPROVED** [review]

**Pull request merged**